### PR TITLE
Validate package.json files for Bassa

### PR DESCRIPTION
**Pull request by kmehant**

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -1,6 +1,16 @@
 {
   "name": "Bassa",
   "version": "1.3.0",
+  "description": "Automated Download Queue for Enterprise to take the best use of Internet bandwidth",
+  "bugs": "https://github.com/scorelab/Bassa/issues",
+  "author": "Malithsen",
```
Comment (malithsen) [inline ui/package.json]: Make this "SCoRe Community"

File: ui/package.json [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+  "contributors": "Malithsen",
```
Comment (malithsen) [inline ui/package.json]: This should contain all the contributors mentioned [here](https://github.com/scorelab/Bassa/graphs/contributors) separated by commas

File: ui/package.json [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+  "repository": "https://github.com/scorelab/Bassa",
+  "keywords": [
+                      "Bassa"
+   ],
+   "licenses" : [],
```
Comment (malithsen) [inline ui/package.json]: GPL-3.0

Comment (kmehant) [reply ui/package.json]: @malithsen sure :D

File: ui/package.json (hunk 2 above)
Comment (kmehant) [reply ui/package.json]: @malithsen sure :)

File: ui/package.json [hunk 4]
```diff
@@ -1,6 +1,16 @@
...
+   "homepage": "http://www.scorelab.org/",
```
Comment (malithsen) [inline ui/package.json]: http://www.scorelab.org/Bassa/

File: ui/package.json [hunk 5]
```diff
@@ -1,6 +1,16 @@
 {
   "name": "Bassa",
   "version": "1.3.0",
+  "description": "Automated Download Queue for Enterprise to take the best use of Internet bandwidth",
+  "bugs": "https://github.com/scorelab/Bassa/issues",
+  "author": "Malithsen",
+  "contributors": "Malithsen",
+  "repository": "https://github.com/scorelab/Bassa",
```
Comment (agentmilindu) [inline ui/package.json]: Following would  be better
```
"repository": { 
   "type" : "git",
   "url" : "https://github.com/scorelab/Bassa.git"
 }
```

**Comment by kmehant** [comment]
@malithsen @agentmilindu Validated the json and also addressed the changes, kindly review :)

**Pull request merged**